src/Nethermind/Nethermind.Test.Runner/StateTestRunner.cs

```diff
@@ -57,11 +59,11 @@ public IEnumerable<EthereumTestResult> RunTests()
             IEnumerable<GeneralStateTest> tests = (IEnumerable<GeneralStateTest>)_testsSource.LoadTests();
             foreach (GeneralStateTest test in tests)
             {
+                if (_filter is not null && !test.Name.StartsWith(_filter))
```

Comment:
Not a regex-like filter with wildcards or test categories?

Reply:
Our codebase adds a some sort of suffix to the `test.Name` not sure what it means. thats why i had to do start with. regex or wildcards would break for the same reason (the suffix)

Reply:
Regex don't break if you add siffuxes, you can still do starts with or contains regex